### ISSUE TITLE
darktable group icons don't match module contents

### ISSUE THREAD

**Issue by PaulSorenson** [darktable-org/darktable#2441]
**description**
The Correction group includes:
- tone curve, zone system local contrast

The tone group includes:
- velvia, color contrast, color correction

The color group includes:
- sharpen, denoise equalizer, 

**To Reproduce**
Open darkroom in darktable. Click on module groups.

**Expected behavior**
modules associated with their rightful group.

**Screenshots**

![image](https://user-images.githubusercontent.com/5038152/56629249-9efea600-6601-11e9-816d-ef061df4d989.png)

**Platform (please complete the following information):**
 - OS: fedora 29 with nvidia drivers
 - 2.7.0+1045 built from source. Note this behaviour is present from earlier versions for some time.

**Additional context**
I did apply a custom base curve to darktable some time ago and it made edits to the database. I couldn't say for sure whether this problem arose from that or whether that is even a plausible cause.

**Comment by Nilvus** [comment]
I didn't have modules in such groups. Velvia for example is in color group (I'm using master branch also). Before I've used the script to change module order in the old GUI. Since the new UI made by Aurelien, this module order is implemented. Maybe you have the old GUI order in your darktablerc config file. When update darktable to the new UI, I've had to revert back this part on classic order. 

See these lines on your darktablerc config file (in your /home/<user>/.config/darktable folder) :

```
plugins/darkroom/group_order/1=1
plugins/darkroom/group_order/2=2
plugins/darkroom/group_order/3=3
plugins/darkroom/group_order/4=4
plugins/darkroom/group_order/5=5
```
If numbers on right are in another order, put them back like my lines (with darktable closed) and launch darktable after save these changes.
